INSTALL.md

```diff
@@ -46,7 +46,7 @@ Prerequisites
 
 To install OpenSSL, you will need:
 
- * A make implementation
+ * A "make" implementation
```

Comment:
Here I deliberately did not use backticks because this is not a code quotation but refers to 'make' in a broader sense, including also, e.g., `nmake`.

Reply:
Very good point!